.github/workflows/skill-drift-check.yml

```
@@ -0,0 +1,34 @@
+jobs:
+  check-drift:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Check skill drift
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2181:style:2:6: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)